docs/quickstart/guide-cargo.md

```diff
@@ -10,11 +10,20 @@ The fastest and easiest way to get MOLGENIS running on a machine, is using our c
 If you are unfamiliar with PostGreSQL, follow one of their [PostGreSQL installation guides](https://www.postgresql.org/docs/9.6/static/index.html). Once you have a PostGreSQL server running, open up the included pgAdmin application that is supplied with most PostGreSQL installations, and perform the following actions:
 
 - Add a database 'molgenis'
+```sql
+CREATE DATABASE molgenis;
+```
```

Comment:
The instructions refer to the pgAdmin tool, do we want them to use command line instead?